src/controllers/services.ts

```diff
@@ -243,3 +260,360 @@ export async function putOrganizationLogo(
     )
     .run();
 }
+
+export async function getReviewStatus(
+  apiClient: ApiManagementClient,
+  jiraClient: IJiraAPIClient,
+  authenticatedUser: AdUser,
+  serviceId: NonEmptyString
+): Promise<
+  | IResponseSuccessJson<ReviewStatus>
+  | IResponseErrorForbiddenNotAuthorized
+  | IResponseErrorConflict
+  | IResponseErrorInternal
+  | IResponseErrorNotFound
+> {
+  const maybeApimUser = await getApimUser(
+    apiClient,
+    authenticatedUser.emails[0]
+  );
+  if (isNone(maybeApimUser)) {
+    return ResponseErrorNotFound(
+      "API user not found",
+      "Cannot find a user in the API management with the provided email address"
+    );
+  }
+  const authenticatedApimUser = maybeApimUser.value;
+
+  // Authenticates this request against the logged in user
+  // checking that serviceId = subscriptionId
+  // if the user is an admin we skip the check on userId
+
+  const maybeSubscription = await getUserSubscription(
+    apiClient,
+    serviceId,
+    authenticatedApimUser.id
+  );
+  if (isNone(maybeSubscription)) {
+    return ResponseErrorNotFound(
+      "Subscription not found",
+      "Cannot get a subscription for the logged in user"
+    );
+  }
+
+  return jiraClient
+    .searchServiceJiraIssue({
+      serviceId
+    })
+    .mapLeft<
+      IResponseErrorConflict | IResponseErrorInternal | IResponseErrorNotFound
+    >(_ => ResponseErrorInternal(_.message))
+    .chain(
+      fromPredicate(
+        _ => _.total > 0,
+        _ =>
+          ResponseErrorNotFound(
+            `Review Status: ${serviceId}`,
+            `There isn't a review for this service ${serviceId}`
+          )
+      )
+    )
+    .chain(_ => taskEither.of(_.issues[0].fields))
+    .fold<
+      // | IResponseSuccessJson<{ readonly status: string }>
+      | IResponseSuccessJson<ReviewStatus>
+      | IResponseErrorForbiddenNotAuthorized
+      | IResponseErrorConflict
+      | IResponseErrorInternal
+      | IResponseErrorNotFound
+    >(identity, card =>
+      ResponseSuccessJson<ReviewStatus>({
+        comment: card.comment,
+        detail: card.status.name,
+        labels: card.labels,
+        status: 200,
+        title: `Review Status: ${card.status} for ${serviceId}`
+      })
+    )
+    .run();
+}
+
+export async function newDisableRequest(
+  apiClient: ApiManagementClient,
+  jiraClient: IJiraAPIClient,
+  authenticatedUser: AdUser,
+  serviceId: NonEmptyString,
+  jiraConfig: config.IJIRA_CONFIG
+): Promise<
+  | IResponseSuccessAccepted
```

Comment:
Seems that `ResponseSuccessAccepted` response never happens in this controller. We can remove it.

Reply:
Fix with a95d264